google/cloud/bigtable/examples/README.md

```diff
@@ -3,25 +3,37 @@
 This directory contains minimal examples to get started quickly, and are
 referenced from the Doxygen landing page as "Hello World".
 
-## Compiling the Examples
+## Table of Contents
+
+- [Before you begin](#before-you-begin)
+  - [Compile the Examples](#compile-the-examples)
+  - [Run the examples against the Cloud Bigtable Emulator](#run-the-examples-against-the-cloud-bigtable-emulator)
+  - [Run the examples against the Cloud Bigtable Service](#run-the-examples-against-the-cloud-bigtable-service)
+- [Samples](#samples)
+  - [Hello World](#hello-world)
+  - [Administer Instances](#administer-instances)
+
+## Before you begin
+
+### Compile the Examples
 
 These examples are compiled as part of the build for the Cloud Bigtable C++
 Client.  The instructions on how to compile the code are in the
 [top-level README](../../README.md) file.
 
-## Running the examples against the Cloud Bigtable Emulator
+### Run the examples against the Cloud Bigtable Emulator
```

Comment:
I think we should remove the sections about running against the emulator.  They won't work for the admin operations, and that is just confusing. @sduskis thoughts?

Reply:
I will fix it after I receive inputs from @sduskis.

Reply:
@sduskis ,
Please suggest...

Reply:
removed this section.